src/ui/osx/TogglDesktop/Feature/TimeEntryEditor/EditorViewController.swift

```diff
@@ -508,7 +508,7 @@ extension EditorViewController: NSTextFieldDelegate {
         guard let textField = obj.object as? NSTextField else { return }
 
         // Duration
-        if textField == durationTextField {
+        if textField == durationTextField && durationTextField.isTextChanged {
```

Comment:
I suggest removing this check as we already perform a check in the method being called here + without this check `controlTextDidEndEditing` method looks more simple and straightforward.